.circleci/config.yml

```diff
@@ -0,0 +1,83 @@
+version: 2.1
+
+orbs:
+    codecov: codecov/codecov@3
+
+jobs:
+    integration-tests:
+        docker:
+            - image: mcr.microsoft.com/dotnet/sdk:7.0
+            - image: rabbitmq:3
```

Comment:
Double Dockering

Reply:
This starts a TCP endpoint for RMQ with default settings. Configuration settings can be overridden in a separate configuration file if needed.